Makefile

```diff
@@ -113,17 +114,20 @@ remove-agents:
 	docker-compose -f monitor-agents/docker/docker-compose.yml down
 
 # Backups
+backup:
+	./scripts/dump.sh postgres-core postgres-events postgres-statutory postgres-discounts
+
 backup_core:
-	./helper.sh --execute postgres-core -- pg_dump 'postgresql://postgres:$${PW_POSTGRES}@localhost/core' --inserts > core.sql.backup-$(shell date +%Y-%m-%dT%H:%M)
+	./scripts/dump.sh postgres-core
 
 backup_events:
-	./helper.sh --execute postgres-events -- pg_dump 'postgresql://postgres:$${PW_POSTGRES}@localhost/events' --inserts > events.sql.backup-$(shell date +%Y-%m-%dT%H:%M)
+	./scripts/dump.sh postgres-events
 
 backup_statutory:
-	./helper.sh --execute postgres-statutory -- pg_dump 'postgresql://postgres:$${PW_POSTGRES}@localhost/statutory' --inserts > statutory.sql.backup-$(shell date +%Y-%m-%dT%H:%M)
+	./scripts/dump.sh postgres-statutory
 
 backup_discounts:
-	./helper.sh --execute postgres-discounts -- pg_dump 'postgresql://postgres:$${PW_POSTGRES}@localhost/discounts' --inserts > discounts.sql.backup-$(shell date +%Y-%m-%dT%H:%M)
+	./scripts/dump.sh postgres-discounts
 
 backup_network:
 	./helper.sh --execute postgres-network -- pg_dump 'postgresql://postgres:$${PW_POSTGRES}@localhost/network' --inserts > discounts.sql.backup-$(shell date +%Y-%m-%dT%H:%M)
```

Comment:
This should be ./scripts/dump.sh postgres-network

Reply:
Ah sorry, didn't notice this addition. Same for below! I will correct in another PR I have for the Makefile